napalm/ios/ios.py

```diff
@@ -97,7 +99,16 @@
 class IOSDriver(NetworkDriver):
     """NAPALM Cisco IOS Handler."""
 
-    def __init__(self, hostname, username, password, timeout=60, optional_args=None):
+    def __init__(
+        self,
+        hostname,
+        username,
+        password,
+        timeout=60,
+        rosetta_driver=None,
```

Comment:
Hmm... what does this argument do? :thinking: I've seen it at the base driver, and assumed that here we pass the name of the NAPALM driver, but now seeing it here, I'm not following anymore (again, I'm not familiar with ntc-rosetta still, and not sure what driver means in that context :sweat_smile:).

Reply:
A driver in the context of rosetta is an object that knows how to parse/translate your device's native configuration/state. Ideally, napalm would figure out which driver to use but you may choose to override it yourself if you want to.